app/guid-file/template.hbs

```diff
@@ -146,58 +146,58 @@
                                 <div data-test-file-language>{{manager.languageFromCode}}</div>
-                                <h2>{{t (concat 'file_detail.metadata_' manager.nodeWord)}}</h2>
-                                <div>
+                                <h2 data-test-metadata-node>{{t (concat 'file_detail.metadata_' manager.nodeWord)}}</h2>
+                                <div data-test-target-funder-div>
                                     {{#each manager.targetMetadata.funders as |funder|}}
-                                        <div>{{t 'file_detail.funder'}}</div>
+                                        <div data-test-target-funder-name-label>{{t 'file_detail.funder'}}</div>
                                         <div data-test-target-funder-name={{funder.funder_name}}>
                                             {{funder.funder_name}}
                                         </div>
-                                        <div>{{t 'file_detail.award_title'}}</div>
+                                        <div data-test-target-funder-award-title-label>{{t 'file_detail.award_title'}}</div>
                                         <div data-test-target-funder-award-title={{funder.funder_name}}>
                                             {{funder.award_title}}
                                         </div>
-                                        <div>{{t 'file_detail.award_uri'}}</div>
+                                        <div data-test-target-funder-award-uri-label>{{t 'file_detail.award_uri'}}</div>
                                         <div data-test-target-funder-award-uri={{funder.funder_name}}>
                                             {{funder.award_uri}}
                                         </div>
-                                        <div>{{t 'file_detail.award_number'}}</div>
+                                        <div data-test-target-funder-award-number-label>{{t 'file_detail.award_number'}}</div>
                                         <div data-test-target-funder-award-number={{funder.funder_name}}>
                                             {{funder.award_number}}
                                         </div>
                                     {{/each}}
                                 </div>
-                                <div>{{t 'file_detail.title'}}</div>
-                                <div data-test-target-title>{{manager.target.title}}</div>
-                                <div>{{t 'file_detail.description'}}</div>
-                                <div data-test-target-description>{{manager.target.description}}</div>
-                                <div>{{t 'file_detail.contributors'}}</div>
+                                <div data-test-file-title-label>{{t 'file_detail.title'}}</div>
+                                <div data-test-file-title>{{manager.target.title}}</div>
```

Comment:
These are incorrect data-test selector descriptions. The metadata here is for the target node, not the file, so they should remain `data-test-target` and not `data-test-file`. Anything below here is still a target item.

Reply:
This has now been corrected by resolving merge conflicts.